apps/timetable/ui/index.html

```diff
@@ -41,7 +41,7 @@
         <div id="gh-page-container">
             <div id="gh-left-container">
                 <div id="gh-meta-container">
-                    <a href="http://cam.ac.uk" id="gh-header-logo" target="_blank" title="Visit the Cambridge University website">
+                    <a href="javascript:;" id="gh-header-logo" target="_blank" title="Visit Timetable">
```

Comment:
These `target="_blank"` can probably go if we're not using the default behaviour anymore?